modules/services/gpg-agent.nix

```
@@ -197,6 +197,30 @@ in {
           now.
         '';
       };
+
+      enableBashIntegration = mkOption {
+        default = true;
+        type = types.bool;
+        description = ''
+          Whether to enable Bash integration.
+        '';
+      };
```

These all can be simplified as follows:

```suggestion
      enableBashIntegration = mkEnableOption "Bash integration" // {
        default = true;
      };
```

---

Ah I was just copy/pasting from some other module I saw. Good to know, thanks!